dev-packages/application-package/src/generator/frontend-generator.ts

```diff
@@ -75,60 +75,99 @@ module.exports = Promise.resolve()${this.compileFrontendModuleImports(frontendMo
 
     protected compileElectronMain(): string {
         return `// @ts-check
-const cluster = require('cluster');
-if (cluster.isMaster) {
-    // Workaround for https://github.com/electron/electron/issues/9225. Chrome has an issue where
-    // in certain locales (e.g. PL), image metrics are wrongly computed. We explicitly set the
-    // LC_NUMERIC to prevent this from happening (selects the numeric formatting category of the
-    // C locale, http://en.cppreference.com/w/cpp/locale/LC_categories).
-    if (process.env.LC_ALL) {
-        process.env.LC_ALL = 'C';
+const path = require('path');
+const { isMaster } = require('cluster');
+const { fork } = require('child_process');
+const electron = require('electron');
+const { app, BrowserWindow, ipcMain } = electron;
+
+// Workaround for https://github.com/electron/electron/issues/9225. Chrome has an issue where
+// in certain locales (e.g. PL), image metrics are wrongly computed. We explicitly set the
+// LC_NUMERIC to prevent this from happening (selects the numeric formatting category of the
+// C locale, http://en.cppreference.com/w/cpp/locale/LC_categories).
```

Comment:
This was is cluster.isMaster before is there a reason it's not anymore ?

Reply:
I don't think it matters much... but it doesn't seem related to the current issue

Reply:
Then let's go with this.

Reply:
I don't want to nitpick but we should avoid adding side changes like that unless necessary otherwise one gets the impression that this change is needed for the fix the work.

Moreover it can introduce bugs as it changes previous behavior.

Reply:
Funny, that we do exactly the opposite with `LC_ALL` than recommended in the ticket from the comment above: https://github.com/electron/electron/issues/9225#issuecomment-300175705

Also, the bug was fixed in `1.6.6` we use `1.7.6`. Finally, someone also mentions, that it should happen very early in the `main.js`, so it is at the wrong place. If I am not deleting it, I will pull up, right above the imports.

Reply:
I have moved it up.

Reply:
Why not remove it if the bug was fixed in our version ?

Reply:
I left the workaround in the code. https://github.com/theia-ide/theia/pull/843/files#r151832774